src/network/networkd-network.h

```diff
@@ -324,6 +324,7 @@ struct Network {
         int ipv4_route_localnet;
         int ipv6_dad_transmits;
         uint8_t ipv6_hop_limit;
+        uint32_t ipv6_retrans_time;
```

Comment:
please use usec_t here, i.e. stick to usecs throughout our codebase, only convert to ms the moment you write it to the sysctl

Reply:
Done.